Examples/Framework/include/ActsExamples/EventData/MuonSimHit.hpp

```diff
@@ -59,8 +60,8 @@ muonMdtIdentifierFields splitId(Acts::GeometryIdentifier::Value theID) {
 Acts::GeometryIdentifier::Value compressId(muonMdtIdentifierFields f) {
```

Comment:
Just for my understanding: how is this being used? Do actually mix these in with "normal" `GeometryIdentifiers`?

Reply:
Yes, it actually calculates from the drift circle parameters (stationeta,phi etc) the geometryId. But is not really necessary for the test, I will replace it in the next commit:)